client/lib/query-manager/test/index.js

```diff
@@ -16,6 +16,11 @@ describe( 'QueryManager', () => {
 
 	beforeEach( () => {
 		manager = new QueryManager();
+		sandbox.create();
```

Comment:
The sandbox isn't new and not being introduced here, it just needs to be `create`/`restored` in the tests because the methods we stub are class methods now. Before, we stubbed an instance method, and got a new "un-stubbed"  instance in the `beforeEach`. Since the class remains, if you stub it, it's stubbed.

It's probably a good practice anyways 🙂